arkouda/strings.py

```diff
@@ -2405,9 +2485,16 @@ class method to return a Strings object attached to the registered name in the a
         Registered names/Strings objects in the server are immune to deletion
         until they are unregistered.
         """
+        from arkouda.pdarrayclass import attach_pdarray
+
         rep_msg: str = cast(str, generic_msg(cmd="attach", args={"name": user_defined_name}))
         s = Strings.from_return_msg(rep_msg)
         s.name = user_defined_name
+        registry = list_registry()
+        if f"{user_defined_name}_bytes" in registry:
+            s._bytes = attach_pdarray(user_defined_name + "_bytes")
+        if f"{user_defined_name}_offsets" in registry:
+            s._offsets = attach_pdarray(user_defined_name + "_offsets")
```

Comment:
Same argument to be made here as `util/attach` from my last review. We're double defining the name for bytes/offsets in 2 places and 2 different ways

Reply:
Updated